### PR TITLE
[RFC] inline docstrings

### PR DESCRIPTION
**Pull request by fredrikekre**
Use docstrings and move it inline.

### CODE REVIEW

**Comment by fredrikekre** [comment]
Do we want to have it like this?
screenshot from 2016-11-07 16-16-40

**Comment by KristofferC** [comment]
Looks very good to me.

**Comment by fredrikekre** [comment]
Review plz!
Built docs: https://fredrikekre.github.io/

**Comment by fredrikekre** [comment]
It would be nice to have an option to not show the module name for the functions. Then we could just have `dot` instead of `Base.LinAlg.dot` in the docs. Does that exist?

**Comment by fredrikekre** [comment]
https://github.com/JuliaDocs/Documenter.jl/issues/253

**Comment by KristofferC** [comment]
Do you think "tensor with order" or "tensor of order" sounds best? From googling it seems maybe "of" is better?

**Comment by fredrikekre** [comment]
of

**Comment by KristofferC** [comment]
`A_{iiii}` is not proper index notation.

File: src/tensor_ops.jl [hunk 1]
```diff
@@ -344,7 +572,10 @@ Computes the symmetric part of a second order tensor, returns a `SymmetricTensor
 end
 
 """
-Computes the (minor) symmetric part of a fourth order tensor, returns a `SymmetricTensor{4}`
+Computes the minor symmetric part of a fourth order tensor, returns a `SymmetricTensor{4}`
+
+```julia
```
Comment (KristofferC) [inline src/tensor_ops.jl:573]: Missing end of code block.

**Comment by fredrikekre** [comment]
I know xD, but how would you write that in index notation? Perhaps just write as a sum?

**Comment by KristofferC** [comment]
Not sure.. probably with some deltas...

**Comment by fredrikekre** [comment]
I cannot find a definition of trace of fourth order tensor...
Should we remove that?

**Comment by KristofferC** [comment]
Yeah, lets do it.

**Comment by KristofferC** [comment]
One day of debugging for this bullshit: https://gist.github.com/KristofferC/6f162620ebcc0300273826f18e591962

**Comment by fredrikekre** [comment]
Merge and see what happens? :)

**Comment by KristofferC** [comment]
go for it

**Comment by KristofferC** [comment]
I dont see travis running at all...

**Comment by fredrikekre** [comment]
No...

**Pull request merged**